src/main/java/model/process/functions/Sum.java

```diff
@@ -26,10 +21,14 @@ public Sum(DataTable model, DataDescriber<NumberValue> argument) {
 	 */
 	@Override
 	public FloatValue calculate() {
+		//TODO decide if this must become default and move to parrent.
```

Comment:
het netje oplossen geeft vrij veel problemen met return values en etc. Dus het kijken naar een nette oplossing hoort denk ik niet in deze branch. Dus vandaar dat ik een tijdelijke fix heb gemaakt.